apps/pwabuilder/src/script/components/app-package-form-base.ts

```diff
@@ -277,9 +277,12 @@ export class AppPackageFormBase extends LitElement {
   private colorChanged(e: UIEvent, formInput: FormInput) {
     const inputElement = e.target as HTMLInputElement | null;
     let formattedValue = (inputElement as unknown as SlColorPicker)!.getFormattedValue('hex').toLocaleUpperCase();
+
+    const colorValue = inputElement?.nextElementSibling;
+    const newValue = document.createElement('p');
+    newValue.textContent = formattedValue;
     
-    let colorValue = inputElement?.nextElementSibling;
-    colorValue!.innerHTML = formattedValue;
+    colorValue!.replaceWith(newValue);
```

Comment:
Looks like you can move everything under the next "if" and in this case you don't need to use non-null assertion. Or you can just check at the beginning of the function:
`if (!inputElement)
  return;
`